### ISSUE TITLE
About the loss scales

### ISSUE THREAD

**Issue by WXDawaken** [experiencor/keras-yolo3#55]
Hi, I am trying to implement yolo3 with mxnet. But I am so confused about the object scale applied to the object loss. I read the source file from the darknet repo, and I found that there is no scale to apply to the object loss, so I did the same thing in my code, which did not work well that the confidence is too small to pass the threshold. 
  And I find that you set the object scale to 5.0 in your [configuration](https://github.com/experiencor/keras-yolo3/blob/25af1ba9f054aacff2f778618116629cc0c370df/config.json#L23). Did you meet the same trouble? or you just want to keep the configuration same to the yolo2? 
  I am so stuck in the loss problem of yolo3. And thank you for your implementation job of yolo2 and yolo3 in keras.
